### ISSUE TITLE
Wiser plugin won't start after update to HS 2024.12 - ModuleNotFoundError: No module named 'aioWiserHeatAPI'

### ISSUE THREAD

**Issue by stuartm** [asantaga/wiserHomeAssistantPlatform#546]
2024.12 updated Python to 3.13 and I suppose this might be related to that? It's far from the only third party integration broken in this release.

```
Logger: homeassistant.setup
Source: setup.py:334
First occurred: 08:25:08 (1 occurrences)
Last logged: 08:25:08

Setup failed for custom integration 'wiser': Unable to import component: No module named 'aioWiserHeatAPI'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/setup.py", line 334, in _async_setup_component
    component = await integration.async_get_component()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1034, in async_get_component
    self._component_future.result()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1014, in async_get_component
    comp = await self.hass.async_add_import_executor_job(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self._get_component, True
        ^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/usr/local/lib/python3.13/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1074, in _get_component
    ComponentProtocol, importlib.import_module(self.pkg_path)
                       ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/util/loop.py", line 200, in protected_loop_func
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.13/importlib/__init__.py", line 88, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 1022, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/config/custom_components/wiser/__init__.py", line 32, in <module>
    from .coordinator import WiserUpdateCoordinator
  File "/config/custom_components/wiser/coordinator.py", line 8, in <module>
    from aioWiserHeatAPI.wiserhub import (
    ...<6 lines>...
    )
ModuleNotFoundError: No module named 'aioWiserHeatAPI'
```

**Comment by msp1974** [comment]
I'm guessing you have not upgraded to latest version as this was picked up in #535

**Comment by stuartm** [comment]
Odd, I had upgraded to the latest available version of all HACS integrations at least couple of days before upgrading to 2024.12. A quick check shows I have the latest version. The Wiser integration wasn't the only one affected missing modules, presumably due to the change in python version and installation directory. Guess this might be a HA or HACS upgrade bug then where modules added during install of integrations were not automatically re-installed during the upgrade process.

I manually installed the missing python module in the HA container and everything is working again.

Closing since this seems like it might be a more general bug and not specific to this integration.
